frontend/src/pages/Exercise/index.js

```diff
@@ -42,14 +67,14 @@ export class Exercise extends Component {
 
   async componentDidMount() {
     await this.props.loadExercises();
-    await this.props.exercise.profile_form.map(item => {
+    await this.props.exercise.profile_form.map(item =>
```

Comment:
`setState` does not return a promise (or anything for that matter) so this statement does not need to be `await`ed

Reply:
ah ofc, thanks

Reply:
ill fix when i hook it up to the back end